core/src/main/java/nl/nn/adapterframework/scheduler/SchedulerFactoryBean.java

```diff
@@ -35,4 +39,13 @@ public void afterPropertiesSet() throws Exception {
 			throw e;
 		}
 	}
+
+	@Override
+	public void setDataSource(DataSource dataSource) {
+		//Make sure this isn't autowired by Spring
+	}
+	@Override
+	public void setTransactionManager(PlatformTransactionManager transactionManager) {
+		//Make sure this isn't autowired by Spring
+	}
```

Comment:
waarom deze twee hier toegevoegd, en uit de config verwijderd?

Reply:
Het zomaar zou kunnen dat iemand custom code oid heeft, en dan alles kapot maakt. Het is ook een voorzorgsmaatregel, als wij Spring autowire ooit aan gaan zetten, dat deze properties niet gezet worden.